schema/workflow.json

```diff
@@ -220,13 +215,13 @@
       "oneOf": [
         {
           "required": [
-            "error",
+            "condition",
```

Comment:
i would not make condition required here so that runtimes that still want to make a logical mapping against the domain-specific error name can do that